app/services/discord.server.ts

```diff
@@ -41,22 +41,12 @@ async function fetchAllGuildMembers(
   }
 }
 
-async function getCachedGuildMembers() {
+export async function isMemberOfGuild(id: string): Promise<boolean> {
   const discordGuildMembers = await redisClient.get("discordGuildMembers");
 
-  if (discordGuildMembers) {
-    return discordGuildMembers;
-  } else {
-    const allGuildMembers = await fetchAllGuildMembers();
-    await redisClient.setEx("discordGuildMembers", 5, allGuildMembers);
-    return allGuildMembers;
-  }
-}
-
-export async function isMemberOfGuild(id: string): Promise<boolean> {
-  const discordGuildMembers = await getCachedGuildMembers();
+  const parsedResult = discordGuildMembers && JSON.parse(discordGuildMembers);
 
-  const parsedResult = JSON.parse(discordGuildMembers);
+  console.log(parsedResult, discordGuildMembers);
```

Comment:
Ideallly we should avoid logging user data unless it has been sanitized before hand

Reply:
This has been removed :)